install/database/2024_04_03_rhessi.sql

```diff
@@ -0,0 +1,189 @@
+INSERT INTO datasources (id, name, description, units, layeringOrder, enabled, sourceIdGroup, displayOrder)
+VALUES
+(95,  'RHESSI', 'RHESSI 3-6keV Back Projection',      NULL, 1, 0, '', 0),
+(96,  'RHESSI', 'RHESSI 6-12keV Back Projection',     NULL, 1, 0, '', 0),
+(97,  'RHESSI', 'RHESSI 12-25keV Back Projection',    NULL, 1, 0, '', 0),
+(98,  'RHESSI', 'RHESSI 25-50keV Back Projection',    NULL, 1, 0, '', 0),
```

Comment:
name column should include what I currently have in description.

This name column is used to show which image is being displayed. Usually this name contains the full selection details.

![image](https://github.com/Helioviewer-Project/api/assets/94071409/725ce7f7-7633-4cbe-91a3-db9b383da1de)

In the above image it shows "RHESSI" from the name column. It should say "RHESSI 3-6keV Back Projection" (hopefully this name isn't too long).

Reply:
The name is too long, for now I'll leave it as is.